src/main/java/org/mskcc/cmo/messaging/model/LinkedPatient.java

```diff
@@ -0,0 +1,53 @@
+package org.mskcc.cmo.messaging.model;
+
+import java.io.Serializable;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Relationship;
+
+/**
+ *
+ * @author ochoaa
+ */
+@NodeEntity
+public class LinkedPatient implements Serializable {
```

Comment:
Curious why this is called LinkedPatient - is there a circumstance when we would have a patient node that is unlinked?

Reply:
I didnt want to get mixed up with the names of classes from the (now) shared cmo-metadb entities repository. 

To answer your question though I dont think there would be a circumstance where a patient node would not be linked.

Reply:
I envisioned the `LinkedPatient` and `LinkedSample` nodes as the way we could support the conversion of sample/patient identifiers between systems (dmp/darwin/etc)